From 4d901fdeb7af70d5e7800848f0a33af6cbeb491d Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 16 Feb 2008 16:22:02 -0500
Subject: [PATCH] Do not run ldconfig for FAKEROOT installs

If you're installing into a temporary directory, then running ldconfig will
simply waste CPU and I/O time.  The install location will not be any path
that ldconfig searches, and generally people build as non-root so the
ldconfig binary will run for a while before erroring out due to lack of
permissions.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 libcap/Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libcap/Makefile b/libcap/Makefile
index fdb9ae2..b108aa9 100644
--- a/libcap/Makefile
+++ b/libcap/Makefile
@@ -62,7 +62,9 @@ install: all
 	install -m 0644 $(MINLIBNAME) $(LIBDIR)/$(MINLIBNAME)
 	ln -sf $(MINLIBNAME) $(LIBDIR)/$(MAJLIBNAME)
 	ln -sf $(MAJLIBNAME) $(LIBDIR)/$(LIBNAME)
+ifeq ($(FAKEROOT),)
 	-/sbin/ldconfig
+endif
 
 clean:
 	$(LOCALCLEAN)
-- 
1.5.4

